### DISCUSSION TITLE
plotting graphs for results analyze.

### DISCUSSION THREAD

**Discussion by soreettihussein** [omnetpp/omnetpp#1153] [Q&A]
Hello everyone
I am trying to understand showcases provided on INET 4.5.0 (TSN). Please anyone can help me how was the comparison of mean end-to-end delay from result directory of FifoQueueing, PriorityQueueing and Frame Preemption graph was plotted. As shown on Figure 1. When was trying to plot graph from this result file it look like this  shown on Figure 2. The way I trying to plot is not, right?
Showcase found at: https://inet.omnetpp.org/docs/showcases/tsn/framepreemption/doc/index.html).
Any suggestions on how to plot graph? 
Thank you in advance.

![Figure 1](https://github.com/omnetpp/omnetpp/assets/163638177/2fc56142-f094-4100-b839-f734c5bd2aa3)

![Figure2](https://github.com/omnetpp/omnetpp/assets/163638177/6dedd6c1-d202-4843-9b96-6769e00841a8)

**Comment by adamgeorge309** [comment]
Can you be more specific on what you are trying to plot? The .anf file in the showcase folder contains all the charts for the showcase. Also, what version of OMNeT++ and INET are you using?

**Reply by soreettihussein** [reply]
@adamgeorge309 
I am trying to compare end-to -end of three traffic shaping mechanism. I want to understand how was the end-to-end of those three mechanisms (FifoQueueing, PriorityQueueing and Frame Preemption) graph was plotted. Because understand of this showcase and other are important for me to go further.  I am using OMNeT++ 6.0.2 and INET 4.5.0.  I am beginner for OMNeT++.
Thank you so much!!!

**Reply by adamgeorge309** [reply]
You can plot this by opening the showcase's .anf file and selecting the meanBitLifeTimePerPacket:vector in the browse data tab. Then select the config names you want to plot (FifoQueueing, PriorityQueueing and Frame Preemption), right-click, and plot. The charts also have a mean vector operation applied to them (right click on chart -> apply -> mean). Also, the .anf file already contains this chart, you can open that more easily.

**Reply by soreettihussein** [reply]
OK thank you so much @adamgeorge309. Can you give me guidance, how can I create new chart for comparison of three scenario!!!
